src/toast/index.ts

```diff
@@ -1,3 +1,15 @@
 export * from "./ToastState";
+export * from "./Toasts";
 export * from "./ToastProvider";
-export * from "./ToastController";
+export * from "./Toasters";
+export * from "./helpers";
+export type {
+  Renderable,
+  ValueFunction,
+  ValueOrFunction,
+  Content,
+  Toast,
+  ToastOptions,
+  DefaultToastOptions,
+  ToastPlacement,
+} from "./ToastTypes";
```

Comment:
Rearrange, looks a bit nicer 

```ts
export * from "./Toasts";
export * from "./helpers";
export * from "./Toasters";
export * from "./ToastState";
export * from "./ToastProvider";
export type {
  Toast,
  Content,
  Renderable,
  ToastOptions,
  ValueFunction,
  ToastPlacement,
  ValueOrFunction,
  DefaultToastOptions,
} from "./ToastTypes";
```

Reply:
I did it already for the types and pushed.